packages/cactus-common/src/main/typescript/logging/logger.ts

```diff
@@ -21,6 +22,61 @@ export interface ILoggerOptions {
   level?: LogLevelDesc;
 }
 
+/**
+ * STANDARD EXCEPTION HANDLING - EXAMPLE:
+ * ====================================
+ * public doSomething(): void {
+ *   try {
+ *     executeSomething();
+ *   } catch (ex) {
+ *     this.log.exception(ex);
+ * }
+ *
+ *
+ *
+ * EXCEPTION HANDLING WITH CONTEXT - EXAMPLE:
+ * ==========================================
+ * public doSomething(): void {
+ *   try {
+ *     executeSomething();
+ *   } catch (ex) {
+ *     this.log.exception(ex, "Some additional information / context about the scenario");
```

Comment:
@m-courtin The re-throw here is missing, similar thoughts as above.

Reply:
@petermetz: Totally agree on the fail-fast principle. Added the re-throw in the documentation of the "Standard Exception Handling" as well as the "Exception Handling with Context"